app/DynamicsAdapter/DynamicsAdapter.Web/Controllers/WeatherForecastController.cs

```diff
@@ -1,7 +1,6 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
-using System.Threading.Tasks;
```

Comment:
Do we really need this file? If yes, can we change the file name?

Reply:
this should not be part of this pr lol, this controller is going to go away anyway, it's just the default controller provided by the boilerplate..

Reply:
I'm alright with removing this as part of the PR

Reply:
not doing anything for this in this PR